test/ffmpeg-vaapi/transcode/transcoder.py

```diff
@@ -170,7 +201,17 @@ def transcode(self):
 
     for n, output in enumerate(self.outputs):
       for channel in xrange(output.get("channels", 1)):
-        encoded = self.goutputs[n][channel]
+        width = output.get("width", 0)
+        height = output.get("height", 0)
+
+        if width > 0 and height > 0:
+          scaled = self.goutputs[n][channel]
+          enc_file = get_media()._test_artifact(
+            "{}_{}_{}".format(self.case, n, channel))
+          encoded = self.scale_to_source_size(scaled, output["codec"], self.mode, self.width, self.height, self.frames, enc_file)
```

Comment:
There is no need to pass `self.*` parameters to member function since you can directly access them from the member function.

Reply:
removed self from variables